integration-libs/opf/checkout/occ/config/default-occ-opf-config.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OccConfig } from '@spartacus/core';
+
+export const defaultOccOpfConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        getActiveConfigurations: 'active-configurations',
```

Comment:
[MEDIUM]

Do we really need to define this endpoint here? It's already defined in the `/base` entry point.

Reply:
[OUTDATED] It is now only part of the base sub-lib.